hack/e2e-suite/pd.sh

```diff
@@ -30,13 +30,13 @@ if [[ "$KUBERNETES_PROVIDER" != "gce" ]] && [[ "$KUBERNETES_PROVIDER" != "gke" ]
   exit 0
 fi
 
-disk_name="e2e-$(date +%H-%M-%s)"
+disk_name="e2e-$(date +%s)"
```

Comment:
%s is already seconds-since-epoch. My suspicion is that the original author
intended to use %S, but %H-%M-%S is actually less unique than just %s alone
(which, if I'm not using $RANDOM like I just did in monitoring.sh, is a
common way I approach uniqueness, too).

On Tue, Jan 13, 2015, 19:38 Alex Robinson <notifications@github.com> wrote:

> In hack/e2e-suite/pd.sh
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3438#discussion-diff-22915051>
> :
>
> > @@ -30,13 +30,13 @@ if [[ "$KUBERNETES_PROVIDER" != "gce" ]] && [[ "$KUBERNETES_PROVIDER" != "gke" ]
> >    exit 0
> >  fi
> >
> > -disk_name="e2e-$(date +%H-%M-%s)"
> > +disk_name="e2e-$(date +%s)"
>
> I'm curious, why this is better than the previous format?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3438/files#r22915051>
> .
>